public/app/plugins/panel/geomap/layers/data/geojsonMapper.ts

```diff
@@ -129,10 +147,20 @@ export const geojsonMapper: MapLayerRegistryItem<GeoJSONMapperConfig> = {
             id: 'config.styles',
             path: 'config.styles',
             name: 'Style Rules',
+            description: 'The style for the first matching rule will apply',
             editor: GeomapStyleRulesEditor,
             settings: {},
             defaultValue: [],
-          });
+          })
+          .addCustomEditor({
+            id: 'config.style',
+            path: 'config.style',
+            name: 'Default Style',
+            description: 'The style to apply when no rules above match',
+            editor: StyleEditor,
+            settings: {},
```

Comment:
I'm guessing we will want to customize both of these StyleEditor instances (at this level and at style rule editor) in a similar way, correct? (i.e. remove field options in favor of fixed, etc.)

Reply:
yes -- I just updated with a *super* simple version